Packs/AzureKubernetesServices/Integrations/AzureKubernetesServices/AzureKubernetesServices.py

```diff
@@ -133,9 +137,13 @@ def clusters_addon_update(client: AKSClient, args: Dict) -> str:
 
 
 def start_auth(client: AKSClient) -> CommandResults:
-    user_code = client.ms_client.device_auth_request()
+    response = client.ms_client.device_auth_request()
+    message = response.get('message')
+    url = re.search(REGEX_SEARCH_URL, message).group("url")  # type:ignore
```

Comment:
`type: ignore` is not the way to go here, as `re.search` might return `None`
lets be more protective here and assert a match as is returned